dt/bgp/kustomization.yaml

```diff
@@ -0,0 +1,175 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  # Set namespace to OpenStack on all namespaced objects without a namespace
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../lib/networking/metallb
+  - networking/bgpmetallb
+  - ../../lib/networking/netconfig
+  - ../../lib/networking/nad
```

Comment:
L21, 23 and 24 can be simplified to `../../lib/networking`

Reply:
Sorry, I was completely wrong about this given the current way `lib/networking` is implemented.  It needs to be reversed to what you had beforehand.

Reply:
ok, thanks. I'm running a d/s job to verify it. I'll let you know how it goes.
(I still have to add the README files)

Reply:
FYI: now either way works, as this [1] has been merged.

[1] https://github.com/openstack-k8s-operators/architecture/pull/196

Reply:
ok, rebased and L21, L23 and L24 replaced by just `../../lib/networking`

I will trigger a d/s job with this change tonight.

Before this change, the PR was tested with d/s jobs and it worked fine (i.e. deployment finished successfully and the setup seems healthy, although we have some connectivity problems between test-operator and provider network IPs, but I don't think changes in this PR would fix it).
Hence, if we get similar results after this change, IMO we could merge this PR (when reviewers approve it, obviously).